### DISCUSSION TITLE
[2주차] - 과제 안내

### DISCUSSION THREAD

**Discussion by JoisFe** [Study-2-Effective-Java/Effective-Java#29] [4. 과제 안내]
# [2주차] - 과제 안내

## 과제 담당
| 아이템 | 담당 |
| --- | --- |
| 아이템 8. finalizer와 cleaner 사용을 피하라 | [ding_cook](https://github.com/bunsung92) |
| 아이템 9. try-finally보다는 try-with-resources를 사용하라 | [coalong](https://github.com/coalong) |
| 아이템 10. equals는 일반 규약을 지켜 재정의하라 | [jinan159](https://github.com/jinan159) |
| 아이템 11. equals를 재정의하려거든 hashCode도 재정의하라 | [JoisFe](https://github.com/JoisFe) |
| 아이템 12. toString을 항상 재정의하라 | [corock](https://github.com/corock) |
| 아이템 13. clone재정의는 주의해서 진행하라 | [gonami-gennnmja](https://github.com/gonami-gennnmja) |
| 아이템 14. Comparable을 구현할지 고려하라 | [chikeem90](https://github.com/chikeem90) |

**Comment by chikeem90** [comment]
@JoisFe 님
제가 아이템14 할게요

**Reply by JoisFe** [reply]
넵 그렇게 해주시면 감사하겠습니다. 수정해두겠습니다.

**Comment by corock** [comment]
제가 @YuDeokRin 님 대신 아이템 12 진행하겠습니다.

**Reply by JoisFe** [reply]
수정해두었습니다.!

**Reply by YuDeokRin** [reply]
감사합니다. ㅠㅠ
